eris.go

```diff
@@ -143,6 +159,14 @@ func (e *rootError) Format(s fmt.State, verb rune) {
 	printError(e, s, verb)
 }
 
+//StackFrames returns a slice of all the frames in the error chain
+func (e *rootError) StackFrames() []uintptr {
+	if e == nil {
```

Comment:
Will this ever happen? I noticed it's not covered in our tests right now (same with the case below)